GP/kernel_modules/kernel_utils.py

```diff
@@ -0,0 +1,14 @@
+# Author: Henry Moss & Ryan-Rhys Griffiths
+"""
+Molecule kernels for Gaussian Process Regression implemented in GPflow.
+"""
```

Comment:
Shall we change this module-level docstring?

Reply:
Probably